src/Compilers/CSharp/Portable/Symbols/NamedTypeSymbol.cs

```diff
@@ -1511,18 +1511,26 @@ internal bool IsTupleTypeOfCardinality(out int tupleCardinality)
 
         // PROTOTYPE: Should be abstract.
         /// <summary>
-        /// Returns an instance of a symbol that has <see cref="IsNativeInt"/> set to <paramref name="asNativeInt"/>
-        /// if the underlying symbol represents System.IntPtr or System.UIntPtr.
+        /// Returns an instance of a symbol that represents an native integer
+        /// if this underlying symbol represents System.IntPtr or System.UIntPtr.
         /// For other symbols, throws <see cref="System.InvalidOperationException"/>.
         /// </summary>
-        internal virtual NamedTypeSymbol AsNativeInt(bool asNativeInt) => throw ExceptionUtilities.Unreachable;
+        internal virtual NamedTypeSymbol AsNativeInteger() => throw ExceptionUtilities.Unreachable;
 
         // PROTOTYPE: Should be abstract.
```

Comment:
>// PROTOTYPE: Should be abstract. [](start = 8, length = 33)

It looks like the comment got lost during the move

Reply:
I dropped the comment intentionally, to match the `IsTupleType` property above it in `TypeSymbol`.

---
In reply to: [385985841](https://github.com/dotnet/roslyn/pull/41970#discussion_r385985841) [](ancestors = 385985841)